README.md

```diff
@@ -374,6 +374,21 @@ DESCOPE_STEP_UP_FLOW_ID=step-up
 # Telemetry key
 DESCOPE_TELEMETRY_KEY=""
 ```
+### Widgets
+Widgets are components that let a Descoper expose management features for tenant-based implementation of B2B(Business To Business).
```

Comment:
I think we need to mention it in another section in readme , not in the example 
In addition 
- i don't think we should use "the descoper" term here, maybe "you"
- i don't think widgets are b2b only


Also worth having a title so next widget have its place , wdyt?

Let's talk about it tomorrow a minute to communicate more freely